src/states/client.rs

```diff
@@ -111,6 +111,13 @@ impl Client {
             Action::Terminate => {
                 return Transition::Terminate.to_evented();
             }
+            Action::ClientAuthority { result_tx } => {
+                let _ = result_tx.send(Authority::Client {
+                    client_key: *self.full_id.public_id().signing_public_key(),
+                    proxy_node_name: *self.proxy_public_id.name(),
+                    peer_id: self.crust_service.id(),
+                });
```

Comment:
Looks like it should be feasible to get the peer ID out of mock Crust without touching production code?
https://github.com/maidsafe/routing/blob/master/src/mock_crust/support.rs#L213